frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -188,7 +188,7 @@ function ReaderRolling:onReadSettings(config)
     --- @fixme remove this branch with migration script
     elseif last_per then
         self.setupXpointer = function()
-            self:_gotoPercent(last_per)
+            self:_gotoPercent(last_per * 100)
```

Comment:
Is there a point in multiplying by 100 here, just to divide by 100 in `ReaderRolling:_gotoPercent`?

Reply:
It's rarely used stuff, and intuitively, a gotoPercent(percent) function would accept a number between 0 and 100 - even if our internal handling expect one between 0 and 1.
We could have the public `ReaderRolling:onGotoPercent(percent)` accept 0-100 and do a `/ 100` and give a 0-1 to the private `ReaderRolling:_gotoPercent(new_percent)`, but it is still named "Percent", so that would be privately confusing :)
But well, rarely used stuff, *100 and then /100 won't cost much.
(I'm sure we have bits in the code with "percent" that required 0-1, but well :)